### PR TITLE
3.5.0 GA cherry picks

### PR DESCRIPTION
**Pull request by akofink**

### CODE REVIEW

**Comment by akofink** [comment]
I plan to look into the regression.

**Comment by akofink** [comment]
The failure seems to be related to https://github.com/Katello/katello/pull/7042. @ehelms would you mind looking into it? Should we not include the cassette change?

**Comment by ehelms** [comment]
[test default]

**Comment by ehelms** [comment]
Looks like maybe rubocop needs to be locked to a particular version in this branch for tests to pass?

**Comment by akofink** [comment]
[test default]

**Comment by akofink** [comment]
The build before Jenkins broke actually had no rubocop failures and one test failure. I'm rerunning the tests to see if maybe that last result was some sort of fluke.

File: katello.gemspec [hunk 1]
```diff
@@ -48,6 +48,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "mocha"
   gem.add_development_dependency "vcr", "< 4.0.0"
   gem.add_development_dependency "webmock"
+  gem.add_development_dependency "rubocop", "~> 0.49.0"
```
Comment (akofink) [inline katello.gemspec]: @ehelms Hmm.. is this not the way to do this? I tested rubocop locally, and this resolves to 0.49.1 which passes.

Comment (ehelms) [reply katello.gemspec]: Oh this is a CI configuration issue due to how I changed to use Github pull request builder. You don't actually need this commit given 1.16-stable locks to 0.49.0

**Comment by ehelms** [comment]
ACK

**Pull request merged**